app/assets/stylesheets/components/page_specific/_category_link_list.scss

```diff
@@ -27,6 +27,10 @@
     // This is to override the highly specific .editorial list styling
     content: none !important;
   }
+
+  > .category-link-list__item__link {
```

Comment:
Cool, I think this is fine in this instance - can't think of a better way to do it, as the more global rule is there to pick up every link within the .editorial section I believe.